diffHistory.js

```
@@ -120,30 +120,39 @@ const getHistories = (modelName, id, expandableFields, cb) => {
         });
 };
 
-const plugin = function lastModifiedPlugin(schema) {
+/**
+ * @param {Object} schema - Schema object passed by Mongoose Schema.plugin
+ * @param {Object} [opts] - Options passed by Mongoose Schema.plugin
+ * @param {string[]} [opts.omit] - Fields to omit from diffs (ex. ['a', 'b.c.d']).
+ */
+const plugin = function lastModifiedPlugin(schema, opts = {}) {
+    if (opts.omit && !Array.isArray(opts.omit)) {
+        delete opts.omit;
```

is this just for validation that omit is an array?

---

If its only for validation, we should throw an error here, instead of silently removing it.